dbc2val/Readme.md

```diff
@@ -104,18 +104,26 @@ A smaller excerpt from the above sample, with less signals.
+|-------------------------------|------------------|-------------------------------|--------------|-----------------------|---------------------------------------------------------------------------------------------------------|
+| config                        | -                | -                             | -            | `--config`           | Configuration file  |
+| dbcfile                       | -                | [can].dbc                     | DBC_FILE     | `--dbcfile`          | DBC file used for parsing CAN traffic |
+| dumpfile                      | -                | [can].candumpfile             | CANDUMP_FILE | `--dumpfile`         | Replay recorded CAN traffic from dumpfile |
+| canport                       | -                | [can].port                    | CAN_PORT     | `--canport`          | Read from this CAN interface  |
+| use-j1939                     | False            | [can].j1939                   | USE_J1939    | `--use-j1939`        | Use J1939. See below for Env var usage |
+| use-socketcan                 | False            | -                             | -            | `--use-socketcan`    | Use SocketCAN (overriding any use of --dumpfile) |
+| mapping                       | mapping/vss_3.1.1/vss_dbc.json | [general].mapping | MAPPING_FILE | `--mapping`        | Mapping file used to map CAN signals to databroker datapoints. Take a look on usage of the mapping file |
+| strict                        | False            | -                             | -            | `--lax-dbc-strict`  | Disable strict parsing of dbc file |
+| server-type                   | kuksa_val_server | [general].server_type         | SERVER_TYPE  | `--server-type`     | Which type of server the feeder should connect to (kuksa_val_server or kuksa_databroker |
+| ip (kuksa_databroker)         | 127.0.0.1        | [kuksa_databroker].ip         | VDB_ADDRESS,<br /> DAPR_GRPC_PORT | -| Databroker IP address, See below for Env var usage |
+| port (kuksa_databroker)       | 55555            | [kuksa_databroker].port       | VDB_ADDRESS,<br /> DAPR_GRPC_PORT | -| Databroker port, See below for Env var usage |
+| protocol (kuksa_databroker)   | grpc             | [kuksa_databroker].protocol   | -            | -                     | Configuration currently ignored |
+| insecure (kuksa_databroker)   | True             | [kuksa_databroker].insecure   | -            | -                     | Configuration currently ignored |
+| ip (kuksa_val_server)         | localhost        | [kuksa_val_server].ip         | -            | -                     | KUKSA.val Server IP address |
+| port (kuksa_val_server)       | 8090             | [kuksa_val_server].port       | -            | -                     | KUKSA.val Server port |
+| protocol (kuksa_val_server)   | ws               | [kuksa_val_server].protocol   | -            | -                     | Configuration currently ignored |
```

Comment:
Currently we ignore "protocol". If server type is "kuksa_val_server" we use ws. Do we actually see a need to be able to specify what protocol to use? 

I see two options:

- Either remove the "protocol" config and let it be implied based on if you use server or broker
- Or keep it but give an error if it does not specify a supported protocol